src/App.js

```diff
@@ -1,13 +1,58 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm'
+import TodoList from './components/TodoComponents/TodoList'
+import './app.css'
 
+
+const todos = []
 class App extends React.Component {
+  constructor () {
+    super();
+    this.state = {
+      todos: todos,
```

Comment:
```suggestion
      todos: []
```

You don't have to set an empty array in memory and then set your state to that empty array, you can just initialize todos as an empty array as shown above.